docs/decisions/0002-lti-1p3-variables.rst

```diff
@@ -0,0 +1,161 @@
+   * - Claim variable
+     - Block location (on course context).
+     - Reasoning
+   * - id
+     - Block location (on course context).
+     - The resource link id needs to be a identifier unique to the deployment.
+       In the Open edX platform, we already have an analog that is unique to the entire instance: block location.
+   * - title
+     - Display name (either from block or parent block)
+       Empty if not available/applicable.
+     - We use the `Display name` attribute to provide a descriptive title to the LTI tool.
+       In case this isn't available in the launch context, this parameter isn't sent.
+   * - description
+     - Not implemented.
+     - This is an optional field.
+
+Reference: http://www.imsglobal.org/spec/lti/v1p3/#resource-link-claim-0
+
+User claims
+~~~~~~~~~~~
+The user data claims are identifiers passed through the LTI message to send user data to the LTI tool.
+
+.. list-table::
+   :widths: auto
+   :header-rows: 1
+
+   * - Claim variable
+     - Open edX
+     - Reasoning
+   * - sub
+     - external user id (with the `lti` type)
+     - Using the id provided by the external_user_id from openedx core.
+       See https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/external_user_ids/docs/decisions/0001-externalid.rst
+   * - name
+     - User full name
+     - User full name as defined in the spec, gated by a feature flag to avoid leaking PII to external tools.
+   * - email
+     - User email address
+     - User email, gated by a feature flag to avoid leaking PII to external tools.
+
+Reference: http://www.imsglobal.org/spec/lti/v1p3/#user-identity-claims-0
+
+
+Context claim
+~~~~~~~~~~~~~
+This claim is optional and includes information about the context where LTI launch is happening.
+
+.. list-table::
+   :widths: auto
+   :header-rows: 1
+
+   * - Claim variable
+     - Open edX
+     - Reasoning
+   * - id
+     - course id if using CourseOffering
+     - This claim requires a unique identifier per deployment,
+       and Course ID is already unique platform wide to represent a course.
+   * - type
+     - http://purl.imsglobal.org/vocab/lis/v2/course#CourseOffering
+     - LTI launches will mostly occur in the context of courses.
+       If launched from outside, the entire context claim should be omitted.
+   * - label
+     - Not used.
+     - This is just a description field that is optional.
+   * - title
+     - `Organization` - `Course name`
+     - Using a readable identifier of the organization and course name to the tool.
+
+Reference: http://www.imsglobal.org/spec/lti/v1p3/#context-type-vocabulary
+
+
+
+Tech Debt
+=========
+
+* The current LTI 1.3 implementation merged on `master` uses the internally generate resource id from the LTI 1.1.1 implementation.
+
+Consequences
+============
+
+* Changing the resource id to the value defined above will make all context on currently deployed XBlocks to be lost (since a different resource means the LTI is being launched
```

Comment:
```suggestion
* Changing the resource id to the value defined above will lose contexts on all currently deployed XBlocks (since a different resource means the LTI is being launched
```